docs/Vector-Similarity-for-RediSearch.ipynb

```diff
@@ -37,40 +37,96 @@
     "\n",
     "Command format:\n",
     "```\n",
-    "FT.CREATE <index_name> SCHEMA <vector field name> VECTOR <type> <dimension> <distance metric> <index algorithm> <algorithm parameters>\n",
+    "FT.CREATE <index_name> SCHEMA <vector field name> VECTOR <type> <num params> [<algorithm parameter name> <parameter value> ...]\n",
     "```\n",
     "### General indexing mandatory parameters\n",
     "Parameters should be given to the index build command in the following order\n",
     "* type - vector data type -  - Currently only `FLOAT32` is supported\n",
     "* dimension - vector dimension.\n",
     "* distance metric - either `L2` for euclidean distance, `IP` for internal product or `COSINE` for cosine similarity should be provided. Note, when `COSINE` is selected the indexed vectors will be normalized upon indexing, and the query vector will be normalized upon query.\n",
-    "* Indexing algorithm - either `BF` for brute force or `HNSW` for HNSW indexing algorithm\n",
+    "* Indexing algorithm - either `FLAT` for brute force or `HNSW` for HNSW indexing algorithm\n",
     "\n",
     "### Brute force (Flat index)\n",
     "This index compares the entire indexed vector data to the query vector and returns the top-k similar vectors, according to the given distance metric.\n",
```

Comment:
With this index, users compare a vector to all other vectors before returning top-k most similar vectors.
Similarity is bases on the distance metric